config/default.yml

```diff
@@ -0,0 +1,11 @@
+toolkit:
+  project_id: project-id
```

Comment:
To keep it consistent shouldn't be `project-id` instead of `project_id`?

Reply:
Yeah that's indeed confusing, but basically configuration has underscores and command options has dashes, it's a convention that we inherited from Robo into the Task Runner, and from the Task Runner in here... :smile:

Reply:
Thanks for the new info, makes sense now. 
 About project_id, it id defined during project request and later during creation of github project and should not be changed, in fact it ideally it should not even exists in code but be deducted from project repository name (DG-**PROJECTID**-REFERENCE is the naming rule for repos). So about use env vars for it, not sure if there is advantage.

Is topic for discussion.

Reply:
Changed.